packages/core/models/split-button.model.ts

```diff
@@ -7,4 +7,8 @@ export interface ISplitButton {
   text: string
   /** ltr | rtl */
   direction?: 'ltr' | 'rtl'
+  /** boolean */
+  menuFitButtonContainer?: boolean
```

Comment:
I thing that is a big big name for a prop, hahaha 🤔 

Maybe just... `isFitMenu`?

Reply:
Made on 5477daf